Doc/library/signal.rst

```diff
@@ -67,10 +67,10 @@ Module contents
    signal (SIG*), handler (:const:`SIG_DFL`, :const:`SIG_IGN`) and sigmask
    (:const:`SIG_BLOCK`, :const:`SIG_UNBLOCK`, :const:`SIG_SETMASK`)
    related constants listed below were turned into
-   :class:`enums <enum.IntEnum>`.
+   :class:`enums <enum.IntEnum>` (respectively :class:`Handlers` and :class:`Sigmasks`).
```

Comment:
I think "respectively" should be last word here: (as Handlers and Sigmasks respectively)